demos/python_demos/face_recognition_demo/README.md

```diff
@@ -0,0 +1,196 @@
+# Interactive Face Recognition Demo
+
+This example demonstrates an approach to create interactive applications
+for video processing. It shows the basic architecture for building model
+pipelines supporting model placement on different devices and simultaneous
+parallel or sequential execution using OpenVINO library in Python.
+In particular, this demo uses 3 models to build a pipeline able to detect
+faces on videos, their keypoints (aka "landmarks"),
+and recognize persons using the provided faces database (the gallery).
+The following pretrained models can be used:
+
+* `face-detection-retail-0004` and `face-detection-adas-0001`, to detect faces and predict their bounding boxes;
+* `landmarks-regression-retail-0009`, to predict face keypoints;
+* `face-reidentification-retail-0095`, to recognize persons.
+
+For more information about the pre-trained models, refer to the [Open Model Zoo](https://github.com/opencv/open_model_zoo/tree/master/intel_models/index.md) repository on GitHub*.
```

Comment:
This should be changed to a relative link. Check the other demos for examples.

Reply:
done